backend/hitas/views/reports.py

```diff
@@ -124,3 +127,58 @@ def list(self, request: Request, *args, **kwargs) -> HttpResponse:
         workbook = build_multiple_ownerships_report_excel(ownerships)
         filename = "Useamman sääntelyn piirissä olevan asunnon omistavat omistajat.xlsx"
         return get_excel_response(filename=filename, excel=workbook)
+
+
+class OwnershipsByHousingCompanyReportSerializer(serializers.ModelSerializer):
+    number = serializers.SerializerMethodField()
+    surface_area = serializers.SerializerMethodField()
+    share_numbers = serializers.SerializerMethodField()
+    purchase_date = serializers.SerializerMethodField()
+    owner_name = serializers.SerializerMethodField()
+    owner_ssn = serializers.SerializerMethodField()
```

Comment:
Try these for the fields:
```
    apartment_number = serializers.IntegerField(source="sale.apartment.apartment_number")
    surface_area = surface_area = HitasDecimalField(source="sale.apartment.surface_area")
    share_numbers = SharesSerializer(source="sale.apartment")  # This can be used unless the share numbers must be pre-concantated already. Not 100% sure if this this source is correct.
    purchase_date = serializers.DateField(source="sale.purchase_date")
    owner_name = serializers.SerializerMethodField()
    owner_ssn = serializers.SerializerMethodField()
```